TestFoundation/TestFileManager.swift

```diff
@@ -8,10 +8,10 @@
 //
 
 #if NS_FOUNDATION_ALLOWS_TESTABLE_IMPORT
-    #if (os(Linux) || os(Android))
-        @testable import Foundation
-    #else
+    #if canImport(SwiftFoundation)
         @testable import SwiftFoundation
+    #else
+        @testable import Foundation
     #endif
 #endif
```

Comment:
Can all of this be moved into `TestImports` with an `#else` for the other bits or does `@testable` need to be in the file that is being tested?

Reply:
I don't know about that, this is mirroring what was there previously.  However, I did have to add it to a couple of files to get the builds to work, so I suspect that we cannot just move these.